src/site/xdoc/server/config-rabbitmq.xml

```diff
@@ -75,23 +75,33 @@
           <dt><strong>mailqueue.view.sliceWindow</strong></dt>
           <dd>
               James divide the view into slices, each slice contains data for a given period, sliceWindow parameter controls this period.
-              This dividing of periods allows faster browsing of the mail queue. Tips for choosing sliceWindow is explained in
+              This dividing of periods allows faster browsing of the mail queue. Tips for choosing sliceWindow are explained in
               <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties">rabbitmq.properties</a>
           </dd>
 
           <dt><strong>mailqueue.view.bucketCount</strong></dt>
           <dd>
               Mails in a mail queue are distributed across the underlying storage service.
               BucketCount describes how to be distributing mails to fit with your James setup
-              Tips for choosing bucketCount is explained in
+              Tips for choosing bucketCount are explained in
               <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties">rabbitmq.properties</a>
           </dd>
 
           <dt><strong>mailqueue.view.updateBrowseStartPace</strong></dt>
           <dd>
               To browse, James needs a starting point and to continuously update that point in runtime.
               UpdateBrowseStartPace describes the probability to update the starting point.
-              Tips for choosing updateBrowseStartPace is explained in
+              Tips for choosing updateBrowseStartPace are explained in
+              <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties">rabbitmq.properties</a>
+          </dd>
+
+          <dt><strong>mailqueue.size.metricsEnabled</strong></dt>
+          <dd>
+              By default, the metrics are enabled for the mail queue size.
+              As computing the size of the mail queue is currently implemented on top of browse operation and thus have a linear complexity,
+              sometimes it can get too big, making it impossible for the ES reporter to handle it correctly without crashing.
+              It can be useful then to disable it.
```

Comment:
I would add that this is a temporary workaround until we get 'getSize' optimlized BTW